### PR TITLE
testing/grafana-image-renderer: new aport

### PR DESCRIPTION
**Pull request by kaey**

### CODE REVIEW

File: testing/grafana-image-renderer/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+pkgdesc="Grafana Backend Plugin that handles rendering of panels & dashboards to PNGs using headless chrome"
+url="https://github.com/grafana/grafana-image-renderer"
+arch="x86_64" # Same as grafana.
+license="Apache-2.0"
+makedepends="yarn"
+depends="nodejs ttf-opensans chromium ca-certificates"
+options="!check" # Has no test suite.
+install="$pkgname.pre-install"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/grafana/$pkgname/archive/v$pkgver.tar.gz
```
Comment (Cogitri) [inline testing/grafana-image-renderer/APKBUILD]: Please don't use $pkgname in the source URL

File: testing/grafana-image-renderer/10-host-param.patch [hunk 2]
```diff
@@ -0,0 +1,29 @@
+diff --git a/src/app.ts b/src/app.ts
```
Comment (Cogitri) [inline testing/grafana-image-renderer/10-host-param.patch]: Is this patch upstream?

Comment (kaey) [reply testing/grafana-image-renderer/10-host-param.patch]: Yes https://github.com/grafana/grafana-image-renderer/pull/40

**Comment by Cogitri** [comment]
Please add that to the patch then so people subsequently working on this have a clue as to where's the patch coming from.

-------- Original Message --------

File: testing/grafana-image-renderer/APKBUILD (hunk 1 above)
Comment (kaey) [reply testing/grafana-image-renderer/APKBUILD]: Done

**Comment by kaey** [comment]
>Please add that to the patch then so people subsequently working on this have a clue as to where's the patch coming from.

Done

**Pull request closed without merging**